packages/spindle-ui/src/Toast/Toast.css

```
@@ -1,67 +1,162 @@
+@import '../IconButton/IconButton.css';
+
 :root {
   --Toast-z-index: 1;
 }
 
 .spui-Toast {
+  box-sizing: border-box;
   left: 0;
+  padding: 0 12px;
   position: fixed;
   right: 0;
   text-align: center;
   z-index: var(--Toast-z-index);
 }
 
-.spui-Toast--content {
-  background-color: var(--color-surface-accent-primary-light);
-  border-radius: 40px;
-  box-shadow: 0px 4.75px 14.25px rgba(8, 18, 26, 0.12);
+.spui-Toast-content {
+  align-items: center;
+  border-radius: 52px;
+  box-shadow: 0px 11px 28px rgba(8, 18, 26, 0.24);
   box-sizing: border-box;
-  color: var(--color-text-accent-primary);
   cursor: default;
-  display: inline-block;
-  font-family: inherit;
-  font-size: 16px;
-  font-weight: bold;
+  display: inline-flex;
   margin: 0;
-  padding: 11px 16px;
+  max-width: 360px;
+  min-height: 48px;
+  padding: 0 16px 0 20px;
 }
 
-.spui-Toast-content--error {
-  background-color: var(--color-surface-caution-light);
-  color: var(--color-text-caution);
-  position: relative;
+.spui-Toast-contentInfo {
+  flex-shrink: 0;
+  font-size: calc(22rem / 16);
```

```suggestion
  font-size: 1.375rem;
```
`.spui-Toast-contentText`のところと`calc`を用いるかどうかが揃っていると良いかなと思いました